cetloxblock/cetloxblock.pyproj.user

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
When you make a `.gitignore` file, this should also be ignored if it isn't already, i.e. `*.pyproj.*` files.